File: programmeerimine/py/yl16.py
from random import *

num = randint(0, 101)
answer = False

while answer == False:
    guess = int(input("paku arv"))
    if guess == num:
        print("õige arv: ")
        answer = True
    elif num < guess:
        print("vastus on väiksem")
    elif num > guess:
        print("vastus on suurem")
        

File: programmeerimine/php/yl16.php
<?php
$num = rand(1, 100);
$game = 1;
while($game == 1){
	$a = 46;
	if($a > $num){
		echo "väiksem";
	}elseif($a < $num){
		echo "suurem";
	}else{
		echo "öige";
	}
	$game = 0;
}
?>
